app/views/posts/index.html.haml

```diff
@@ -1,5 +1,4 @@
-- @posts.each do |post|
-  = render 'posts/post', post: post
+= render partial: 'posts/post', collection: @posts
```

Comment:
i think `render @posts` will also do that job instead of this line

Reply:
no , it doesn't , it render the STI view like links/link not posts/post